core/scheduler/metrics.go

```diff
@@ -57,7 +57,7 @@ var (
 		Subsystem: "scheduler",
 		Name:      "validator_balance_gwei",
 		Help:      "Total balance of a validator by public key",
-	}, []string{"pubkey_full", "pubkey"})
+	}, []string{"pubkey_full", "pubkey", "status"})
```

Comment:
note that this increases cardinality since the label "status" can take on atmost 10 values

Reply:
`Total balance of a validator by public key by status`